css/main.css

```diff
@@ -0,0 +1,49 @@
+@import url(typography.css);
+@import url(../resources/css/normalize.css);
+@import url(utils.css);
+
+textarea {
+  font-variant: normal;
+}
+
+body {
+  color: var(--secondary-color);
+}
+
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.main-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.margin-left {
+  margin-left: 28px;
+}
+
+.font-color {
+  color: var(--cta-color);
+}
+
+.banner-styled {
+  font-size: 80px;
```

Comment:
using rem will be good

Reply:
will update rem in second pr for component library